daemon/language.go

```diff
@@ -14,7 +14,14 @@ func GetLanguage(filename string, content []byte) string {
 		return lang
 	}
 
-	lang = strings.ToLower(lang)
+	return normalize(lang)
+}
+
+// normalize maps enry language names to the bblfsh ones.
+// TODO(bzz): remove this as soon as language aliases are supported in bblfsh
+// driver manifest.
+func normalize(languageName string) string {
+	lang := strings.ToLower(languageName)
 	lang = strings.Replace(lang, " ", "-", -1)
 	lang = strings.Replace(lang, "+", "p", -1)
 	lang = strings.Replace(lang, "#", "sharp", -1)
```

Comment:
This is an interesting way of solving this :)

Do you think making it an explicit map makes sense?

Reply:
> This is an interesting way of solving this :)

Well, it's the smallest possible change consistent with current behavior :)

As this is a workaround, I would prefer to keep this change as least invasive as possible. 

Given that we do not have much tests for it and that the current approach already works (TM) for API calls \wo the language -  I would postponed changing implementation until proper alias support.